electrum/base_wizard.py

```diff
@@ -142,11 +142,23 @@ def new(self):
             ('imported', _('Import external watch-only BTCV addresses or private keys')),
         ]
 
+        link = "https://translations.bitcoinvault.global/pdf/BTCV_Tutorial/BTCV-ShortTutorial-2Pager-en.pdf"
+        h_txt = "Learn more"
+        hint = ' '.join([
+            _("<b>2-Key Vault</b><br/>"),
```

Comment:
It would be more readable if formatting chars had been put outside i18n function call. Moreover use single quote `'` for single line, instead of triple of them.